app/src/org/commcare/views/media/MediaLayout.java

```diff
@@ -350,13 +351,13 @@ private void setupInlineVideoView(String inlineVideoURI) {
      * It's height is always 0(from LayoutInspector) even if you set it to match_parent.
      */
     private void makeVideoViewVisible() {
-        int[] maxBounds = getMaxCenterViewBounds();
         //These surprisingly get re-jiggered as soon as the video is loaded, so we
         //just want to give it the _max_ bounds, it'll pick the limiter and shrink
         //itself when it's ready.
+        DisplayMetrics metrics = this.getContext().getResources().getDisplayMetrics();
         ViewGroup.LayoutParams params = videoView.getLayoutParams();
-        params.width = maxBounds[0];
-        params.height = maxBounds[1];
+        params.width = metrics.widthPixels;
+        params.height = metrics.heightPixels;
```

Comment:
the reason why videoView doesn't appear after download completion was this maxBounds[1]. In my case, the textView had a really long text and I had to scroll down to see the videoView. 

The `getMaxCenterViewBounds` method subtracts the text height from the total height
`maxHeight = maxHeight - textViewContainer.getChildAt(0).getHeight();` making this `maxBounds[1]` to be 0. 

Since, videoView will automatically configure itself with the min height possible, we're good to give it max height here.